app/src/main/java/org/p2p/wallet/home/MainFragment.kt

```diff
@@ -175,7 +191,25 @@ class MainFragment :
         super.onDestroyView()
     }
 
+    private fun navigateFromDeeplink(data: DeeplinkData) {
+        Timber.e("ITEM_ID ${R.id.historyItem}")
+        Timber.d("Navigate from deeplink: ${data.target}")
```

Comment:
you can use Timber.i if you want to create a log entry in Sentry / crashlytics

Reply:
yeah, there's just a testing logs, forgot to remove